client/knowledge_base_search/src/containers/SearchContainer/styles.scss

```diff
@@ -1,39 +1,76 @@
+@import 'watson-ui-components/dist/lib/core/_index';
+@import 'watson-ui-components/dist/lib/components/tab-panels/_tab-panels';
+@import 'watson-react-components/dist/scss/components/text-input/_text-input';
+
 .header--input {
   color: #ffffff;
   text-align: center;
   margin-bottom: 2rem;
 }
 
-.tab-panels {
-  background-color: transparent;
-  border: none;
-}
-
-.tab-panels--tab.base--a {
-  color: #a7fae6;
-  font-weight: 200;
-  margin-right: 2rem;
-  border-bottom-width: 0.2rem;
-  padding: 0.5rem 0rem;
-}
-
-.tab-panels--tab.base--a:hover {
-  border-bottom-color: transparent;
-}
-
-.tab-panels--tab.active {
-  color: #fff;
-  border-bottom-color: #fff;
-}
-
-.tab-panels--tab.base--a:focus {
-  color: #fff;
-  border-bottom-color: #fff;
-}
-
-.tab-panels--tab-content {
-  background: inherit;
-  padding: 2rem 0 0 0;
+.search_container--section {
+  .tab-panels {
+    background-color: transparent;
+    border: none;
+  }
+
+  .tab-panels--tab.base--a {
+    color: #a7fae6;
+    font-weight: 200;
+    margin-right: 2rem;
+    border-bottom-width: 0.2rem;
+    padding: 0.5rem 0rem;
+  }
+
+  .tab-panels--tab.base--a:hover {
+    border-bottom-color: transparent;
+  }
+
+  .tab-panels--tab.active {
+    color: #fff;
+    border-bottom-color: #fff;
+  }
+
+  .tab-panels--tab.base--a:focus {
+    color: #fff;
+    border-bottom-color: #fff;
+  }
+
+  .tab-panels--tab-content {
+    background: inherit;
+    padding: 2rem 0 0 0;
+  }
+
+  button.white--button,
+  button.view_all--button {
+    color: #ffffff;
+    border-color: #ffffff;
```

Comment:
nitpick. Sometimes there is short-hand `#fff` and sometimes there is long-hand `#ffffff`

Reply:
okay ima gonna use the `color` function